source/onboard/mattermost-limits.rst

```diff
@@ -36,13 +38,6 @@ Mattermost Free limits
    
    Mattermost reserves the right to archive or delete messages and establish or change (1) limits as to how many messages can be stored, (2) how long such messages will be stored, and (3) charges for storing such messages. Mattermost further reserves the right to establish or change limits on periods of inactivity that may result in the termination of your messages cloud storage and deletion or archiving of any stored messages.
 
-Mattermost Professional limits
```

Comment:
@johndavidlugtu - Can you please confirm that this line should be removed from the docs please?

Reply:
I believe we would no longer have Professional file limits, or at least we won't be enforcing the 250GB file storage limit. @esethna to verify

Reply:
@cwarnermm based on [checklist](https://docs.google.com/spreadsheets/d/1mjXnrPuunAiE9QIcVldOvX2s6_Ih49uUs5OHoFwxzc8/edit#gid=1538757693&range=B19) we'll continue to have a Professional file limit, but not enforce it. Note that this is for Cloud only.

Deferring to you how/if we want to surface this

Reply:
Actually @cwarnermm it looks like the limit is gone for Cloud Pro altogether.

Reference: https://docs.google.com/presentation/d/11hYF5p5sZ_mBi3lF1eT3tHJSM6Iw4uOFkhQG-VK5uPc/edit?disco=AAAAnYSbYJA

Reply:
Apologies for the back and forth!